src/request_collector/cli.py

```diff
@@ -2,22 +2,49 @@
 
 monkey.patch_all(subprocess=False, thread=False)  # isort:skip # noqa
 
+import os.path
+import sys
+
 import click
 import structlog
 from request_collector.server import RequestCollector
 
 from monitoring_service.database import SharedDatabase
+from raiden.utils.cli import NetworkChoiceType
 from raiden_libs.cli import common_options
 
 log = structlog.get_logger(__name__)
 
 
 @click.command()
+@click.option(
+    "--chain-id",
+    type=NetworkChoiceType(["mainnet", "ropsten", "rinkeby", "goerli", "kovan", "<NETWORK_ID>"]),
```

Comment:
Nice! Is there a way to reuse the helpstring, too?

Reply:
Don't think so, but we can make it a constant and use that in the future.